client/sidebar.jsx

```diff
@@ -91,7 +87,7 @@ class Sidebar extends React.Component {
       return (
         <div className="sb-greyBottomBorder">
           <div className="sb-bigSpace sb-floatLeft">
-            <img src={ShipQuickly} alt="" />
+            <img src="https://live.staticflickr.com/65535/50230580552_b385a5e7a6_t.jpg" alt="" />

```

Comment:
Maybe divide these img tags so that they're indented across multiple lines? I'm not sure, but I think the lines have to be less than 80 characters.